tests/metrics/utils/test_genai_metrics.py

```diff
@@ -8,11 +8,12 @@
 from mlflow.exceptions import MlflowException
 from mlflow.gateway.schemas.completions import Candidate, Metadata, ResponsePayload
 from mlflow.metrics.base import EvaluationExample
-from mlflow.metrics.utils import model_utils
+from mlflow.metrics.utils import correctness, model_utils
 from mlflow.metrics.utils.make_genai_metric import _format_variable_string, make_genai_metric
+from mlflow.metrics.utils.prompts.v1 import CorrectnessMetric
 
 # Example properly formatted response from OpenAI
-properly_formatted_openai_response = ResponsePayload(
+properly_formatted_openai_response1 = ResponsePayload(
     candidates=[
         Candidate(
             text='{\n  "Score": 3,\n  "Justification": "The provided output mostly answers the '
```

Comment:
thanks for making this code more DRY. one more thing - can we use the new `openai_justification1` string in here (with the appropriate formatting)?